WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -43,29 +43,54 @@ struct CartView: View {
 /// View sub-components
 ///
 private extension CartView {
+    private var checkoutButtonDisabled: Bool {
+        return viewModel.itemsInCart.isEmpty
+    }
```

Comment:
Suggestion: How about moving this logic to the `PointOfSaleDashboardViewModel`?
```
    var isCheckoutEnabled: Bool {
        itemsInCart.isNotEmpty
    }
```
That way we can unit test the bool value if we add or remove products to the cart, and the view doesn't know anything about the cart state, just to update itself when changes.

Reply:
Good point!